README.md

```diff
@@ -24,6 +24,8 @@ This Git repo will remain available, in case anyone is interested in forking it
 
 There are several other extensions available that include similar features. Feel free to send pull requests editing this README to list your extension here.
 
+- A custom take on this extension - [Quick View for Google Calendar](https://chrome.google.com/webstore/detail/quick-view-for-google-cal/jjmfhjhfkipdojpiohplpfoliolcemlp).
```

Comment:
Can you please add this at position 2? Most users might prefer the exact copy. Thank you!